static/include/robot/apis/generated/robot.md

```diff
@@ -406,7 +422,7 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 **Example:**
 
 ```go {class="line-numbers linkable-line-numbers"}
-status, err := machine.Status(ctx)
+status, err := machine.Status(ctx, nil)
```

Comment:
why ctx here and not context.Background()? I thought we had decided to always do that

Reply:
oh context as ctx is everywhere in the docs, I normally use context.Background but thought we had decided to do that (ctx). I saw it in machine management ticket as well but didn't think there was a problem. Could we make a separate ticket to address sweepingly?

Reply:
sorry, I was just confused. Will do. @npentrel